doc/rose-rug-task-run.html

```diff
@@ -745,12 +745,19 @@ <h2 id="rose-task-run.util.rose_bunch">Built-in Application:rose_bunch</h2>
 
       <dd>Optional (default=continue). Specify what action you want the job to
       take on failure of a command it is trying to run. If set to
-      <var>continue</var> it will run all command variants . If set to
-      <var>abort</var> then on failure of one of the command variants it will
-      stop trying to run any new variants N.B. the job will wait for any
-      already running commands to finish before exiting. Commands that won't be
-      run due to aborting will be reported in the job output with a
-      <var>[SKIP]</var> prefix when running in verbose mode.</dd>
+      <var>continue</var> all command variants will be run by the job and the
```

Comment:
It was the previous line and re-reading it after your much improved following change to the text I think the words "the" and "that" are missing, "Specify what action you want the job to take on `the` failure of a command `that` it is trying t run."